python/tank/deploy/tank_commands/validate_config.py

```diff
@@ -113,30 +246,139 @@ def _run(self, log):
         log.info("")
         log.info("")
         log.info("")
-        
-        
 
+    def _dump(self, log, envs, params):
+        """
+        Dump the supplied environments with the specified parameters
+
+        :param log: A logger instance.
+        :param envs: list of environment objects
+        :param params: parameter dict.
+        """
+
+        if params["dump_sparse"]:
+            dump_type = environment.UpdateAllSettingsFormat.SPARSE
```

Comment:
I think I need to remove their use in this file. They're not really needed here. I will address.